democracy/views/section_comment.py

```diff
@@ -49,6 +57,13 @@ class SectionCommentSerializer(BaseCommentSerializer):
     """
     Serializer for comment added to section.
     """
+    label = JSONPKRelatedField(
+                queryset=Label.objects.all(),
```

Comment:
Weird indentation. (The kwargs should be indented 8 spaces; the trailing `)` should be flush with `label = `.)